happex/db.py

```diff
@@ -57,37 +57,68 @@ def get_2018_countries(self):
         cur.execute("SELECT country FROM happy WHERE year=2018 ORDER BY country;")
         rows = cur.fetchall()
         cur.close()
-        return rows
+        countries = []
+        for item in rows:
+            countries.append(item[1])
+        return countries
```

Comment:
This method of flattening a list causes an entire unnecessary allocation, which can cause performance issues because the list is quite large. Itertools provides a method of flattening a list more efficiently: `itertools.chain.from_iterable(rows)`. You can do `from itertools import chain`, then `chain.from_iterable` for brevity.

Reply:
Using itertools here is unnecessary;  Not only is the maximum output only going to be a max of 250 lines (which isn't really an issue for modern computers to loop through) the documentation for itertools show that it works almost identically to what I have, and identically in terms of computational complexity: https://docs.python.org/2/library/itertools.html

As such I've left this the same.